src/constants/feature-lists-data.ts

```diff
@@ -309,19 +309,19 @@ const featureListData = {
         items: [
           {
             content:
-              "Amplify provides advanced, intuitive data modeling capabilities. Define your API using our the schema.graphql file in the CLI or Amplify Studio's visual editor. Model relationships between types, customize fields, and configure validation rules. Amplify handles turning the schema into fully implemented backend and frontend code.",
+              "Model relationships between types, customize fields, and configure validation rules using the CLI or Amplify Studio's visual editor. Amplify turns your schema into fully implemented backend and frontend code. ",
             linkText: 'Powerful data modeling',
             link: 'build-a-backend/graphqlapi/data-modeling/'
           },
           {
             content:
-              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Subscriptions allow real-time data sync. Integrate backend data sources like DynamoDB easily.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Sync data in real time and integrate backend data sources easily.',
             linkText: 'Seamless real-time data access',
             link: 'build-a-backend/graphqlapi/subscribe-data/'
           },
           {
             content:
-              'Implement fine-grained authorization rules and policies to securely access data. Control auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
```

Comment:
This is not the right text for "Granular authorization." Should be: Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.